components/automate-ui/src/app/pages/+compliance/+reporting/reporting.component.spec.ts

```diff
@@ -68,6 +68,18 @@ describe('ReportingComponent', () => {
     expect(qe).not.toBeNull();
   });
 
+  it('convertMomentToDateWithoutTimezone', () => {
```

Comment:
These new tests are, I understand, specifically to cover the bug. But consider expanding them with a few other key values, typically around boundaries. There's a test helper `using` in spec-helpers.ts, that makes it easy to test all equivalence classes:
```
  using([
    [23, 'an hour before end of day'],
    [0, 'exactly end of day'],
    [1, 'an hour after end of day']
  ], function (hour: number, description: string) {
    it(`maps hour correctly without timezone for ${description}`, () => {
      const dateBefore = moment(`20191023-${hour}00`, 'YYYYMMDD-HHMM');
      const dateAfter = component.convertMomentToDateWithoutTimezone(dateBefore);
      expect(dateBefore.hour()).toEqual(dateAfter.getHours());
    });
  });
```

Reply:
Thank you. The using block is a nice tool.